### PR TITLE
doc: Add new issues templates

### PR DESCRIPTION
**Pull request by remyleone**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+<!--- Please list the affected resources and data sources. --->
+
+* scaleway_XXXX
+
+### Terraform Configuration Files
+
+<!--- Information about code formatting: https://help.github.com/articles/basic-writing-and-formatting-syntax/#quoting-code --->
+
+```hcl
+# Copy-paste your Terraform configurations here - for large Terraform configs,
```
Comment (kindermoumoute) [inline .github/ISSUE_TEMPLATE/bug-report.md:44]: IMHO we should add  something like `Copy-paste the minimal (if possible) terraform configuration to reproduce the bug here`

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+# please use a service like Dropbox and share a link to the ZIP file. For
+# security, you can also encrypt the files using our GPG public key: https://keybase.io/hashicorp
+```
+
+### Debug Output
+
+<!---
+Please provide a link to a GitHub Gist containing the complete debug output. Please do NOT paste the debug output in the issue; just paste a link to the Gist.
```
Comment (kindermoumoute) [inline .github/ISSUE_TEMPLATE/bug-report.md:52]: link to gist?

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+
+### Steps to Reproduce
+
+<!--- Please list the steps required to reproduce the issue. --->
+
+1. `terraform apply`
+
+### Important Factoids
+
+<!--- Are there anything atypical about your accounts that we should know? For example: Running in EC2 Classic? --->
```
Comment (kindermoumoute) [inline .github/ISSUE_TEMPLATE/bug-report.md:77]: EC2?

File: .github/ISSUE_TEMPLATE/feature-request.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+# security, you can also encrypt the files using our GPG public key.
+```
+
+### References
+
+<!---
+Information about referencing Github Issues: https://help.github.com/articles/basic-writing-and-formatting-syntax/#referencing-issues-and-pull-requests
+--->
+
+* #0000
```
Comment (kindermoumoute) [inline .github/ISSUE_TEMPLATE/feature-request.md:43]: ?

Comment (remyleone) [reply .github/ISSUE_TEMPLATE/feature-request.md:43]: Referencing issue #42 with "#42"

Comment (kindermoumoute) [reply .github/ISSUE_TEMPLATE/feature-request.md:43]: Gotcha

**Comment by kindermoumoute** [comment]
LGTM

**Pull request merged**